docs/data/data-grid/getting-started/getting-started.md

```diff
@@ -60,7 +60,9 @@ pnpm add @mui/styled-engine-sc styled-components
 
 </codeblock>
 
-Take a look at the [Styled engine guide](/material-ui/guides/styled-components/) for more information about how to configure `styled-components` as the style engine.
+:::info
+Take a look at the [Styled Engine guide](/material-ui/guides/styled-engine/) for more information about how to configure `styled-components` as the style engine.
```

Comment:
The link change is probably a leftover from older commits.
https://mui.com/material-ui/guides/styled-engine/ redirects to https://mui.com/material-ui/guides/styled-components/ anyway.

```suggestion
Take a look at the [Styled Components guide](/material-ui/guides/styled-components/) for more information about how to configure `styled-components` as the style engine.
```